src/main/java/io/zksync/wallet/DefaultZkASyncWallet.java

```diff
@@ -259,33 +261,39 @@ public EthereumProvider createEthereumProvider(Web3j web3j, ContractGasProvider
     }
 
     @Override
-    public CompletableFuture<Boolean> enable2FA() {
+    public CompletableFuture<Boolean> enable2FA(@Nullable String pubKeyHash) {
```

Comment:
Enabling 2FA is full (no specfiic treatment for a pubKeyHash). I know, there is the same typescript type used by both enabling the 2FA and disabling it. It is a bit misleading.

TL;DR: the pubKeyHash should only be for disabling 2FA